src/Rings/mpoly-graded.jl

```diff
@@ -1162,7 +1162,7 @@ base_ring(f::MPolyDecRingElem) = base_ring(forget_decoration(f))
 
 function show_homo_comp(io::IO, M)
   (W, d) = get_attribute(M, :data)
-  n = get_attribute(W, :name)
+  n = AbstractAlgebra.find_name(W)
   io = pretty(io)
   if n !== nothing
     print(io, LowercaseOff(), "$(n)_$(d.coeff) of dim $(dim(M))")
```

Comment:
```suggestion
    print(io, LowercaseOff(), "$(n)_")
    print(IOContext(io, :supercompact => true), d)
    print(io, " of dim $(dim(M))")
```
Should work too and would be more consistent regarding commas in the degree.

Reply:
I would like this PR to get going fast. There will be a follow up once https://github.com/Nemocas/AbstractAlgebra.jl/pull/1594 is released anyway, where I will incorporate this change